setup.json

```diff
@@ -86,7 +86,7 @@
             "PyCifRW~=4.4",
             "ase~=3.18",
             "matplotlib~=3.3,>=3.3.4",
-            "pymatgen>=2019.7.2,<=2022.02.03,!=2019.9.7",
+            "pymatgen>=2019.7.2,!=2019.9.7",
```

Comment:
Pymatgen does not follow semantic versioning, not sure whether it is a good idea to leave this unbounded. Any future release could break our package.

Reply:
Added `<=2022.1.9` which is the latest version and for which our tests pass.